consensus/service/src/api/client_api_service.rs

```diff
@@ -58,6 +66,7 @@ impl ClientApiService {
         is_serving_fn: Arc<(dyn Fn() -> bool + Sync + Send)>,
         authenticator: Arc<dyn Authenticator + Send + Sync>,
         logger: Logger,
+        tracked_sessions: Arc<Mutex<LruCache<ClientSession, ClientSessionTracking>>>,
```

Comment:
How will LRUCache be used ?

Reply:
The intent is to use this to keep track of how many failed transaction proposals a client has sent recently, so that it can drop connections with clients that look broken / malicious.

Reply:
got it, thank you! :)